strawberryfields/backends/states.py

```diff
@@ -1559,9 +1559,10 @@ def is_coherent(self, mode, tol=1e-10):
         Returns:
             bool: True if and only if the state is a coherent state.
         """
-        mu, cov = self.reduced_gaussian([mode])  # pylint: disable=unused-variable
-        cov /= self._hbar / 2
-        return np.allclose(cov, np.identity(2), atol=tol, rtol=0)
+        # mu, cov = self.reduced_gaussian([mode])  # pylint: disable=unused-variable
+        # cov /= self._hbar / 2
+        # return np.allclose(cov, np.identity(2), atol=tol, rtol=0)
```

Comment:
```suggestion
```

Reply:
For methods that I did not rewrite yet for this backend, I am leaving the original code, but commented-out, in case it is useful for future development.